contrib/dart/.github/workflows/publish.yml

```diff
@@ -3,12 +3,20 @@ name: Publish to pub.dev
 on:
   push:
     tags:
-      - 'v[0-9]+.[0-9]+.[0-9]+*'
+      - "v[0-9]+.[0-9]+.[0-9]+*"
 
 jobs:
   publish:
+    name: "Publish to pub.dev"
     permissions:
-      id-token: write # Required for authentication using OIDC
-    uses: dart-lang/setup-dart/.github/workflows/publish.yml@v1
-    # with:
-    #   working-directory: path/to/package/within/repository
\ No newline at end of file
+      id-token: write # This is required for requesting the JWT for publishing
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: dart-lang/setup-dart
+      - name: Install dependencies
+        run: dart pub get
+      - name: Publish - dry run
+        run: dart pub publish --dry-run --skip-validation
```

Comment:
This is needed, because pub enforces a linear version history. At the moment, the latest release on pub.dev for the network client is 1.6.2 and the latest release we're trying to release is 1.20. This trips that validation.

But we might run into this again, because the version is managed by the Ory Network, and if publishing to pub.dev fails again, we would need to skip the validation again. So leaving this here indefinitely, is probably the best solution.